docs/source/md/lung_model.md

```diff
@@ -0,0 +1,61 @@
+
+This model is designed to perform segmentation of CT scans of human torsos. It is trained to identify 5 key structures: left lung, right lung, heart, spinalcord and esophagus.
+
+## Terms of Use
+
+This model is intended for research purposes only. You are responsible for the performance, the necessary testing, and if needed any regulatory clearance for any of the models produced by this toolbox.
+
+
+## Download
+
+The lung segmentation model can be downloaded from [this release](https://github.com/microsoft/InnerEye-DeepLearning/releases/tag/v0.8).
+
+## Connected Components
+
+It is possible to apply connected components as a post-processing step, and by default this is performed on the 3 largest structures: both lungs and the heart. To alter this behaviour, update the property `largest_connected_component_foreground_classes` of the Lung class in `InnerEye/ML/configs/segmentation/Lung.py`.
+
+## Model Card
+
+### Model Details
+
+- Organisation: Biomedical Imaging Team at Microsoft Research, Cambridge UK.
+- Model date: 31st October 2022.
+- Model version: 1.0.
+- Model type: ensemble of 3D UNet. Training details are as described in [this paper](https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2773292).
+- Training details: 5 fold ensemble model. Trained on the [LCTSC 2017 dataset](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=24284539) (described in detail below).
+- License: The model is released under MIT license as described [here](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/LICENSE).
+- Contact: innereyeinfo@microsoft.com.
+
+### Limitations
+
+The dataset used for training contains only 60 scans, 10 of which are withheld for testing. This limited amount of training data means that the model may not yet generalise well to data samples from outside the dataset and underperforms on the smaller structures (esophagus and spinalcord).
+
+### Intended Uses
+
+This model is intended for research purposes only. It is intended to be used as a starting-point for more challenging segmentation tasks or training using more thorough and comprehensive segmentation tasks.
+
+### Training Data
+
+This model is trained on the [LCTSC 2017 dataset](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=24284539). For a detailed description on this data, including the contouring guidelines, see [this page](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=24284539#242845396723d79f9909442996e4dd0af5e56a30).
+
+To create the dataset used for training, we utilised the [InnerEye-CreateDataset tool](https://github.com/microsoft/InnerEye-CreateDataset). The dataset was processed using the following command:
+
+```shell
+  .\InnerEye.CreateDataset.Runner.exe dataset --datasetRootDirectory=C:\Users\a-phessey\Datasets\lung_dicom --niftiDatasetDirectory=lung_nifti --dicomDatasetDirectory=LCTSC --geoNorm 1 1 3 --groundTruthDescendingPriority esophagus spinalcord lung_r lung_l heart
+```
+
```

Comment:
Add here that the `lung_nifti` folder needs to be uploaded to Azure afterwards and registered.